From: Sebastian Noack <sebastian.noack@gmail.com>
Date: Wed, 4 Oct 2017 22:39:24 -0400
Subject: Statically link libjson-c

Newer Debian releases provide libjson-c.so.3 with the libjson-c3 package,
while older Debian releases provide libjson-c.so.2 with the libjson-c2 package.
So in order to keep this package compatible with a wide range of Debian releases,
we link libjson-c statically.
---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8648d26..6bd31da 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -49,6 +49,8 @@ if (LD_LIBRARY_PATH)
 endif()
 
 find_package(JsonC 0.12.1 REQUIRED)
+set(JSONC_LIBRARIES libjson-c.a)
+
 find_package(PCRE REQUIRED)
 find_package(Wayland REQUIRED)
 find_package(XKBCommon REQUIRED)
